### DISCUSSION TITLE
Cost

### DISCUSSION THREAD

**Discussion by petersimard** [knaufinator/6DOF-Rotary-Stewart-Motion-Simulator#18] [Q&A] [answered]
Could I get the cost of putting together this sim?

**Comment by knaufinator** [comment, marked as answer]
about $5k usd for my own build, most of which is tied up in the motors and gears + shipping for them. the base + frame is relatively inexpensive when welding from raw metal stock... if you use aluminum extrusion this would be more though which a few have gone down this way.
